docs/output.md

```diff
@@ -612,6 +612,21 @@ Whole genome annotation is the process of identifying features of interest in a
 
 </details>
 
+### MetaEuk
+
+In cases where eukaryotic genomes are recovered in binning, [MetaEuk](https://github.com/soedinglab/metaeuk) is also available to annotate eukaryotic genomes quickly with standards-compliant output files.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `Annotation/MetaEuk/[assembler]/[bin]/`
+  - `[bin].fas`: fasta file of protein sequences identified by MetaEuk
+  - `[bin].codon.fas`: fasta file of nucleotide sequences corresponding to the protein sequences fasta
+  - `[bin].headersMap.tsv`: tab-separated table containing the information from each header in the fasta files
+  - `[bin].gff`: annotation in GFF3 format
```

Comment:
No sample information in there?

Reply:
It’s in [bin] but could be made more explicit!